### ISSUE TITLE
Allow editors to rename merged submissions at any time

### ISSUE THREAD

**Issue by soytakyon** [SoylentNews/rehash#442]
Editors should be given a way to edit the title of (only) merged submissions at any time. This will allow us to put a descriptive title/headline in place instead of the merge date+time, prevent merges from festering overlooked in the submissions list, and allow us to merge items on sight without having to post it, preventing more submissions from being posted without being merged.

**Comment by Fnord666b** [comment]
Agreed.  This is quite annoying.  Could also allow the editor to set the merged title at the time of merge.
